crates/swift-bridge-ir/src/parse/parse_extern_mod/function_attributes.rs

```diff
@@ -177,13 +177,17 @@ mod tests {
 
                     #[swift_bridge(return_into)]
                     fn some_function () -> Foo;
+
+                    fn print_hello_world();
                 }
             }
         };
 
         let module = parse_ok(tokens);
 
         assert!(module.functions[0].return_into);
+        assert_eq!(module.functions.len(), 2);
```

Comment:
What's this? Seems like this should be removed?

Reply:
Sorry...I usually add some test-codes to understand how `swift-bridge` works. I forgot to remove this. From now on, I'll remove them before sending a PR.